backend/de.metas.fresh/de.metas.fresh.base/src/main/sql/postgresql/ddl/functions/BusinessPartnerAccountSheetReport_Union.sql

```diff
@@ -0,0 +1,89 @@
+/*
+ * #%L
+ * de.metas.fresh.base
+ * %%
+ * Copyright (C) 2025 metas GmbH
+ * %%
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as
+ * published by the Free Software Foundation, either version 2 of the
+ * License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public
+ * License along with this program. If not, see
+ * <http://www.gnu.org/licenses/gpl-2.0.html>.
+ * #L%
+ */
+
+DROP FUNCTION IF EXISTS BusinessPartnerAccountSheetReport_Union(p_c_bpartner_id numeric,
+                                                                p_dateFrom      date,
+                                                                p_dateTo        date,
+                                                                p_ad_client_id  numeric,
+                                                                p_ad_org_id     numeric,
+                                                                p_isSoTrx       TEXT,
+                                                                p_ad_language   text)
+;
+
+
+CREATE OR REPLACE FUNCTION BusinessPartnerAccountSheetReport_Union(p_c_bpartner_id numeric,
```

Comment:
i don't understand why this function is needed. why not changing `BusinessPartnerAccountSheetReport` function and accept null p_IsSOTrx parameter?

i.e. change the BusinessPartnerAccountSheetReport function and
* when u insert into `temp_BusinessPartnerAccountSheetReport`, also introduce IsSOTrx column
* change all the code below and instead of using p_IsSOTrx use temp_BusinessPartnerAccountSheetReport.IsSOTrx

temp_BusinessPartnerAccountSheetReport